src/actions/authActions.js

```diff
@@ -0,0 +1,52 @@
+import { REGISTER_USER, LOGIN_USER, LOGOUT_USER } from "@constants/actionTypes";
+
+export const registerUser = (user) => {
+  return (dispatch) => {
+    dispatch({
+      type: REGISTER_USER,
+      payload: user,
+    });
+    const existingUsers = JSON.parse(localStorage.getItem("users")) || [];
+    existingUsers.push(user);
```

Comment:
Let's name it `users` for simplicity. Making a variable named `existingUsers` means there are non existing users as well.
```suggestion
    const users = JSON.parse(localStorage.getItem("users")) || [];
    uesrs.push(user);
```

Reply:
done